cmake/modules/AddSwift.cmake

```diff
@@ -722,10 +722,15 @@ endfunction()
 # This is a temporary workaround until it's possible to compile libswift with
 # cmake's builtin swift support.
 function(add_libswift name)
+  set(libswift_compile_options
+      "-target" "x86_64-apple-macosx10.15" # TODO: this is a hack until I figure out where the target is being set.
```

Comment:
This won't be part of the final PR. Also note: it's not an issue with #38675 because we don't eagerly try to import things we shouldn't. Landing that patch is a better long-term solution.

Reply:
Actually I'm going to leave this here for now until that PR lands.